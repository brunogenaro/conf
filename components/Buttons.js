import React, { PureComponent } from 'react';

import Button from './Button';

export default class extends PureComponent {
  render() {
    return (
      <div className='cont'>
        <Button label="Early Birds" pattern='/static/media/images/btn1Pattern'/>
        <Button label="Newsletter" pattern='/static/media/images/btn1Pattern'/>
        <style jsx>{`
            .cont {
              text-align: center;
              margin: 30px;
              > h2: {
                margin: 0;
                letterSpacing: 6;
              }
              @media(max-width: 720px): {
                margin: 5px;
              }
            }
          `}
        </style>
      </div>
    );
  }
}
