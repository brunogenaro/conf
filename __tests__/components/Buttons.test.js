import React from 'react';
import renderer from 'react-test-renderer'
import Buttons from '../../components/Buttons.js';

describe('With Snapshot Testing', () => {
  it('Buttons Snapshot Testing', () => {
    const component = renderer.create(<Buttons />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot()
  })
});