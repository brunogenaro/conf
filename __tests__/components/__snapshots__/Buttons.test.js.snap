// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With Snapshot Testing Buttons Snapshot Testing 1`] = `
<div
  className="cont"
  data-jsx={2911285025}
>
  <div
    className="background"
    data-jsx={2406450419}
  >
    <div
      className="cont"
      data-jsx={2406450419}
    >
      <h2
        data-jsx={2406450419}
      >
        Early Birds
      </h2>
    </div>
  </div>
  <div
    className="background"
    data-jsx={2406450419}
  >
    <div
      className="cont"
      data-jsx={2406450419}
    >
      <h2
        data-jsx={2406450419}
      >
        Newsletter
      </h2>
    </div>
  </div>
</div>
`;
